kfs-core/src/main/resources/edu/arizona/kfs/sys/businessobject/datadictionary/Building.xml

```diff
@@ -0,0 +1,63 @@
+<beans xmlns="http://www.springframework.org/schema/beans"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p"
+	xmlns:dd="http://rice.kuali.org/dd"
+	xsi:schemaLocation="
+http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.0.xsd
+http://rice.kuali.org/dd http://rice.kuali.org/dd/dd.xsd">
+
+	<bean id="Building" parent="Building-parentBean">
+		<property name="attributes">
+			<list merge="true">
+				<ref bean="Building-routeCode" />
+			</list>
+		</property>
```

Comment:
Need to add:
property name="inactivationBlockingDefinitions"
     list merge="true"
             bean parent="InactivationBlockingDefinition"   p:blockedReferencePropertyName="extension.routeCodeObj"